exonum/src/api/public/blockchain_explorer.rs

```diff
@@ -134,10 +132,21 @@ impl ExplorerApi {
     }
 
     fn transaction_info(&self, hash: &Hash) -> Result<TransactionInfo, ApiError> {
-        if let Some(tx) = self.pool.read().expect("Unable to read pool").get(hash) {
-            Ok(TransactionInfo::InPool {
-                content: tx.serialize_field().map_err(ApiError::InternalError)?,
-            })
+        let snapshot = self.blockchain.snapshot();
+        let schema = Schema::new(&snapshot);
+        if schema.transactions_pool().contains(hash) {
```

Comment:
IMO, it could make sense to augment the explorer functionality with mempool-related functions. Can be done later though.

Reply:
Can you show an example of "mempool-related functions" that can be add into explorer?